projects/wave-core-new/src/lib/layers/legend/legend-vector/vector-legend-component.html

```diff
@@ -1,13 +1,87 @@
-<div fxLayout="column">
-    <div fxLayout="row" fxLayoutAlign="start center" class="segment" *ngIf="showDefaultStyle">
+<mat-card fxLayout="column">
```

Comment:
I guess we should leave it as a div since we don't add the additional layer of *depth* there with drop-shadows.

![image](https://user-images.githubusercontent.com/4947061/127994953-003b4e2a-7a69-4094-b0e9-9422d6299879.png)


![image](https://user-images.githubusercontent.com/4947061/127994855-a626df10-bcf5-4e70-b63e-c769f0fb4272.png)

Reply:
I changed it back: 3d5f966e761ff9e0888ee8ff23d48968641d47e2